src/v2/controllers/FormController.ts

```diff
@@ -247,7 +247,10 @@ export default Controller.extend({
       // if Util.isAndroidOVEnrollment() returns true we use a user gesture to complete the redirect in AutoRedirectView
       if (!Util.isAndroidOVEnrollment()) {
         const currentViewState = this.options.appState.getCurrentViewState();
-        Util.redirectWithFormGet(currentViewState.href);
+        // OKTA-702402: redirect only if/when the page is visible
+        Util.executeOnVisiblePage(() => {
+          Util.redirectWithFormGet(currentViewState.href);
+        });
```

Comment:
Do we want to do this at all times? Or only when we are in a chrome browser?

Reply:
cc: @lesterchoi-okta

Reply:
AFAICT the normal active tab behavior will not be affected, and this should handle the inactive tab scenario for other supported browsers that have an "inactive tab" feature -- Edge, Firefox, Safari. Is that correct?

Reply:
Correct